Parsers/ASimWebSession/Parsers/ASimWebSessionApacheHTTPServer.yaml

```diff
@@ -0,0 +1,76 @@
+Parser:
+  Title: Web Session ASIM parser for Apache HTTP Server
+  Version: '0.1'
+  LastUpdated: Jun 19, 2023
+Product:
+  Name: Apache HTTP Server
+Normalization:
+  Schema: WebSession
+  Version: '0.2.6'
+References:
+- Title: ASIM Web Session Schema
+  Link: https://aka.ms/ASimWebSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Apache HTTP Server 
+  Link: https://httpd.apache.org/docs/2.4/logs.html
+Description: |
+  This ASIM parser supports filtering and normalizing Apache HTTP Server logs produced by the Microsoft Sentinel Apache connector to the ASIM Web Session normalized schema.
+ParserName: ASimWebSessionApacheHTTPServer
+EquivalentBuiltInParser: _ASim_NetworkSession_ApacheHTTPServer
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let Parser=(disabled:bool=false){
+    ApacheHTTPServer_CL
+    | where not(disabled)
+    | project  RawData, TimeGenerated, Computer, _ResourceId, Type, _ItemId
+    | where RawData matches regex @'^((25[0-5]|(2[0-4]|1\d|[1-9]|)\d)\.?\b){4}.*\[.*\]\s\"(GET|HEAD|POST|PUT|DELETE|CONNECT|OPTIONS|TRACE|PATCH)\s\S*\s\S*"\s\d{3}\s\d+\s".*\"\s".*\"'
+    | parse RawData with SrcIpAddr " " ClientIdentity " " SrcUserName "[" Date '] "' HttpRequestMethod " " Url " " Protocol '" ' EventResultDetails " " ReturnedObjectSize ' "' HttpReferrer '" "' HttpUserAgent '"' *
+    | project-away RawData
+    | parse _ResourceId with * "/subscriptions/" DvcScopeId "/" *
+    | project-rename 
+        DvcHostname    = Computer,
+        DvcId          = _ResourceId,
+        EventUid       = _ItemId
+    | extend 
+        HttpVersion    = tostring(split(Protocol,"/")[1]),
+        EventStartTime = TimeGenerated,
+        EventEndTime   = TimeGenerated,
+        Dates          = split(replace_string(tostring(split(Date, " ")[0]), @"/", " "), ":"),
+        DvcIdType      = iff (DvcId == "", "", "AzureResourceID")
+    | extend 
+        EventTimeGenerated = todatetime(
+            strcat(
+                Dates[0], 
+                " ", 
+                Dates[1],
+                ":", 
+                Dates[2],
+                ":",
+                Dates[3]
+            )
+        )
+    | extend 
+        HttpStatusCode = EventResultDetails,
+        UserAgent      = HttpUserAgent,
+        EventResult    = iff (
+          toint(EventResultDetails) < 400, "Success", 
+          "Failure"
```

Comment:
When EventResultDetails is empty, that is due to the fact that the message is not in the expected format. EventResult will be failure. In my test 33 out of 126297 records have the incorrect format. Reasons are "Gnutella Connect" because it contains connect and some script injection attempts.